src/help/renderer/HelpApp.vue

```diff
@@ -38,73 +94,6 @@
 	</div>
 </template>
 
-<script>
-import MdiWindowClose from 'vue-material-design-icons/WindowClose.vue'
-import NcButton from '@nextcloud/vue/dist/Components/NcButton.js'
-import NcTextArea from '@nextcloud/vue/dist/Components/NcTextArea.js'
-import { translate as t } from '@nextcloud/l10n'
-
-import { appData } from '../../app/AppData.js'
-
-export default {
-	name: 'HelpApp',
-
-	components: {
-		MdiWindowClose,
-		NcButton,
-		NcTextArea,
-	},
-
-	inheritAttrs: false,
```

Comment:
Not needed here anymore?

For myself: https://github.com/vuejs/rfcs/blob/master/active-rfcs/0040-script-setup.md#declaring-additional-options

Reply:
It was here by mistake. Added a "long" time ago, then Help was implemented as a `NcModal` wrapper.

In SFC Setup it's easy to add it via
```js
defineOptions({
  inheritAttrs: false,
})
```

But in Vue 2 it is not available and requires an additional block:
```html
<script>
export default {
  inheritAttrs: false, 
}
</script>
```